hermes/src/db.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::Result;
+
+pub mod local_cache;
+
+pub type UnixTimestamp = u64;
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub struct DbRecord {
+    pub time:  UnixTimestamp,
+    pub value: Vec<u8>,
+}
+
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum RequestTime {
+    Latest,
+    FirstAfter(UnixTimestamp),
+}
+
+/// This trait defines the interface for a database. The design can support both current
+/// batch-based attestation and future attestation based on accumulators. A accumulated
+/// data contains all the state at a single timestamp; but as we will probably have different
+/// accumulators, we store each data on a separate key. The data is defined as a binary
```

Comment:
i don't understand the sentence starting with "a accumulated data". I think a better way to say this would be something like: "If an item should appear under multiple keys (e.g., a batch of price updates), then ..."